src/ray/core_worker/context.h

```diff
@@ -34,6 +34,11 @@ class WorkerContext {
 
   const ActorID &GetCurrentActorID() const;
 
+  /// Returns whether we are in a direct call actor.
+  bool CurrentActorIsDirectCall() const;
+
+  /// Returns whether we are in a direct call task. This encompasses both direct
+  /// actor and normal tasks.
   bool CurrentTaskIsDirectCall() const;
```

Comment:
The reason is that an actor "taints" the worker with being direct call for all future tasks. However a task should not, since the actor creation task is not a direct call (yet...). It's kind of confusing but it's only temporary while we support raylet calls.